daphne/src/protocol/aggregator.rs

```diff
@@ -81,15 +81,13 @@ pub trait EarlyReportState {
 
 /// Report state during aggregation initialization after consuming the report share. This involves
 /// decryption as well a few validation steps.
-#[derive(Clone, Serialize, Deserialize)]
-#[serde(rename_all = "snake_case")]
+#[derive(Clone)]
 pub enum EarlyReportStateConsumed {
     Ready {
         metadata: ReportMetadata,
-        #[serde(with = "serialize_bytes")]
         public_share: Vec<u8>,
-        #[serde(with = "serialize_bytes")]
         input_share: Vec<u8>,
+        draft_latest_payload: Option<Vec<u8>>,
```

Comment:
What is this value, and why is it only set in the latest draft?

Reply:
Ah, right this is "initial prep message". Consider giving it a more meaningful name, like "draft_latest_prep_init_payload".

In any case, add a code comment explaining what this is and why it's not needed in draft02.